### ISSUE TITLE
Documentation not clear enough for non-pros?

### ISSUE THREAD

**Issue by CharlieIGG** [bsideup/rx-connect#1]
Hello Guys, I've spent an hour looking at rx-connect and it's motivations, and all sounds awesome enough that I want to try it... however I'm an amateur programmer who has only made a couple of React and React Native apps and I cannot wrap my mind around how to exactly implement it... is it possible for you to provide a more "ELI5" documentation?
Also, could this work with React-Native?

**Comment by bsideup** [comment]
Hi @CharlieIGG !

Thanks for your feedback :) If you made a couple of React and React Native apps then you're already pretty advanced cool developer, don't be shy :)

However, since this project is mostly about RxJS, I understand that it might be difficult to understand it for one who don't know it yet. 

Good starting point is not related to RxConnect, but RxJS. 
Watch some courses on Egghead: https://egghead.io/technologies/rx
Plus you can find a lot more of information on the internet

Regarding RxConnect itself - I would really appreciate if you can say what you expect to see in "ELI5"-style documentation :) 

Together we can make it better, and don't hesitate - we're all pioneers in a modern JS world!

**Comment by bsideup** [comment]
@CharlieIGG 

I know that learning something is better when you have something you already know and rewrote version of it. I found WeatherApp repository in your GitHub account and decided to modify it with RxConnect :D 

https://github.com/CharlieIGG/ReactWeatherApp/pull/1

As you can see, both `src/components/App.jsx` and `src/wComponents/App.jsx` are using rxConnect now, so you can diff your version and RxConnect and make some assumptions how it help to transform the code. Have fun! :)

**Comment by CharlieIGG** [comment]
Thank you @bsideup! that is awesome, I will immerse myself a little bit more with your refactor of the WeatherApp and more RxJS documentation and touch back with documentation suggestions. ;)
